lib/pbench/test/functional/agent/cli/commands/tools/test_tools_list.py

```diff
@@ -41,6 +92,17 @@ def tool_with_random_file(self, pbench_run):
         tool = p / "perf"
         tool.touch()
 
+    @pytest.fixture
+    def tools_on_multiple_hosts(self, pbench_run):
+        p = pbench_run / "tools-v1-default" / "testhost.example.com"
+        p.mkdir(parents=True)
+        for tool in ["perf", "mpstat"]:
+            (p / tool).touch()
+        p = pbench_run / "tools-v1-default" / "testhost2.example.com"
+        p.mkdir(parents=True)
+        for tool in ["iostat", "sar"]:
+            (p / tool).touch()
```

Comment:
Would it make for a more interesting test if there were a partial overlap between the hosts' tool-sets?